src/components/pages/home/index.tsx

```diff
@@ -149,19 +178,19 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
         </section>
 
         <section className="mt-20 sm:mt-24">
-          <FeatureRow resource={awsFeatures} />
+          {awsFeatures && <FeatureRow resource={awsFeatures} />}
         </section>
 
         <section className="mt-20 sm:mt-24">
-          <FeatureRow resource={cssFeatures} />
+          {cssFeatures && <FeatureRow resource={cssFeatures} />}
         </section>
 
         <section className="mt-20 sm:mt-24">
-          <FeatureRow resource={reduxFeatures} />
+          {reduxFeatures && <FeatureRow resource={reduxFeatures} />}
         </section>
 
         <section className="mt-20 sm:mt-24">
-          <FeatureRow resource={typescriptFeatures} />
+          {typescriptFeatures && <FeatureRow resource={typescriptFeatures} />}
```

Comment:
I wonder if you could take this and 
```
 const typescriptFeatures = find(homePageData?.homePageFeatures?.features, [
    'slug.current',
    'typescript-features',
  ])
```

and combine them into a component that takes in a string like `'typescript-features'` as props

Reply:
finding the data by slug is something I want to address in a later PR. It would be nice if the front end didn't care at all what actual data it received